### PR TITLE
review time

### PR DESCRIPTION
**Pull request by Gebort**
We need to add this file to the main branch asap

### CODE REVIEW

**Comment by qwmks** [comment]
Delete entire windows folder, just to be sure

File: reviewFile.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+We need to review this file!
```
Comment (qwmks) [inline reviewFile.txt:1]: *You need to

File: reviewFile.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+if (true) {
+
+	deleteFile("system32");
```
[the remark this answers is not in the document]
Comment (Gebort) [reply reviewFile.txt:5]: alright

File: reviewFile.txt (hunk 1 above)
Comment (Gebort) [reply reviewFile.txt:1]: ok

**Pull request merged**